tests/python/utils.py

```diff
@@ -404,7 +404,7 @@ def exec_nvfuser(
         *,
         new_fusion_expected=True,
         device=None,
-        is_clonable=False,
+        is_clonable=True,
```

Comment:
What is the motivation for switching this default value -- is it just so we run the cloned fusion check for all tests instead of a few selected ones?

Reply:
Yes, all the tests are clonable after this PR.